src/ostorlab/runtimes/local/runtime.py

```diff
@@ -360,13 +360,26 @@ def _start_jaeger_service(self):
             self._log_streamer.stream(self._jaeger_service.service)
 
     def _check_services_healthy(self):
-        """Check if the rabbitMQ service is running and healthy."""
-        if self._mq_service is None or self._mq_service.is_healthy is False:
+        """Check if the core services are running and healthy."""
+        return self._are_services_ready()
+
+    @tenacity.retry(
+        stop=tenacity.stop_after_attempt(20),
+        wait=tenacity.wait_fixed(0.5),
+        retry_error_callback=lambda lv: lv.outcome,
```

Comment:
I think this line can be replace with `re_raise=True`

Reply:
It will be done in another PR for all the similar checks.